### DISCUSSION TITLE
Does OTD support the touch on the CTH/CTT models

### DISCUSSION THREAD

**Discussion by KoleckOLP** [OpenTabletDriver/OpenTabletDriver#3164] [General]
I think it does not and if it does that I don't know how to enable it,

I was thinking that maybe I could uninstall the wacom driver and never use it again since windows ink works for drawing and absolute mode works for osu.

**Comment by jamesbt365** [comment]
OTD does not support touch output on any tablet at the moment. See #1664.

**Reply by KoleckOLP** [reply]
okay I assumed so, thank you for the response.
